docs/guides/advanced_user/hydromt_python.rst

```diff
@@ -79,29 +76,60 @@ You can find more detailed examples on using the DataCatalog and DataAdapter in
-- :ref: flw <flw_api>: Hydrological methods for raster DEM data. For example, calculate flow direction, flow accumulation, stream network, catchments, or reproject hydrography.
-- :ref: gis_utils <gis_utils_api>: other general GIS methods. For example to compute the area of a grid cell or to perform a merge of nearest geodataframe.
-- :ref: stats <statistics>: Statistical methods including ``skills`` to compute skill scores of models (e.g. NSE, KGE, bias and many more) and ``extremes`` to analyse extreme events (extract peaks or compute return values).
+FlowPy Wrappers:
+----------------
 
-The available workflows in HydroMT are:
+`PyFlwDir <https://deltares.github.io/pyflwdir/latest/index.html>` contains a series of
+methods to work with gridded DEM and flow direction datasets. The ``gis.flw`` module
+builds on top of this and provides hydrological methods for raster DEM data. For
+example, calculate flow direction, flow accumulation, stream network, catchments, or
+reproject hydrography.
+
+.. _stats:
+
+Stats:
+------
 
-- :ref: grid <workflows_grid_api>: generic workflows to prepare regular gridded data. Used with the ``GridModel``. For example to prepare regular grid data from constant, from RasterDataset (with or without reclassification) or from GeoDataFrame.
-- :ref: mesh <workflows_mesh_api>: generic workflows to prepare unstructured mesh data. Used with the ``MeshModel``. For example to create a mesh grid or prepare unstructured mesh data from RasterDataset.
-- :ref: basin_mask <workflows_basin_api>: workflows to prepare a basin mask based on different region definitions (bounding box, point coordinates, polygon etc.)
-- :ref: rivers <workflows_rivers_api>: workflows to prepare river profile data like width and depth.
-- :ref: forcing <workflows_forcing_api>: workflows to prepare meteorological forcing data. For example to prepare precipitation, temperature, or compute evapotranspiration data.	Advanced downscaling methods are also available within these workflows.
+The ``stats`` module has statistical methods including ``skills`` to compute skill
+scores of models (e.g. NSE, KGE, bias and many more) and ``extremes`` to analyse extreme
+events (extract peaks or compute return values).
+
+.. _processes:
+
+The ``model`` module has a ``processes`` submodule. This module contains some functions
+to work with different kinds of model in- and ouput.
+
+* :ref:`grid <workflows_grid_api>`: generic workflows to prepare regular gridded data.
+    Used with the ``GridModel``. For example to prepare regular grid data from constant,
+    from RasterDataset (with or without reclassification) or from GeoDataFrame.
+* :ref:`mesh <workflows_mesh_api>:` generic workflows to prepare unstructured mesh
+    data. Used with the ``MeshModel``. For example to create a mesh grid or prepare
+    unstructured mesh data from RasterDataset.
```

Comment:
We don't have `GridModel` or `MeshModel` anymore, just Models which may or may not have grid or mesh models